web/Web/web/interfaces/CAL/functions.py

```diff
@@ -79,9 +79,10 @@ def add_session(session, seed_query, mode):
             'seed_query': seed_query,
             'judgments_per_iteration': 1,
             'async': True,
-            'mode': mode}
+            'mode': mode,
+            'seed_judgments': ','.join([j[0]+':'+str(j[1]) for j in judgments])
```

Comment:
I'm not sure if having `seed_judgments` empty will break things, but let's only include it if the list isn't empty
```
if seed_judgments:
    body['seed_judgments'] = ','.join([j[0]+':'+str(j[1]) for j in judgments]
```


Can you update `add_session()` to use the `request` library? I want to move from using `httplib2` to `requests`. I'll update the other functions later.